site/_includes/layouts/author-individual.njk

```diff
@@ -0,0 +1,62 @@
+{% from 'macros/cards/frontmatter-card.njk' import frontMatterCard with context %}
+{% from 'macros/icon.njk' import icon with context %}
+
+{% extends "layouts/base.njk" %}
+
+{% block title_bar %}
+{% include 'partials/breadcrumbs-author.njk' %}
+{% endblock %}
+
+{% block content %}
+<div class="gap-top-300 lg:gap-top-400 pad-left-400 pad-right-400">
+  <div class="center-margin project-hero text-align-center">
+    {% Img
+      src=paged.image,
+      alt=paged.title | i18n(locale),
+      height="192",
+      width="192",
+      class="rounded-full display-inline-block",
+      sizes="(min-width: 481px) 192px, 128px",
+      options={minWidth: 128}
+    %}
+    <h1 class="lg:gap-top-400 type--h4">{{ paged.title | i18n(locale) }}</h1>
+    <p class="type gap-top-300">{{ paged.description | i18n(locale) | safe }}</p>
```

Comment:
How many people's descriptions have HTML in them? Or is this just so my shruggie works? 😆

Reply:
None, I think this is some copypasta from web.dev